explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -526,21 +516,21 @@ getGenesisSummary
     :: ExplorerMode ctx m
     => m CGenesisSummary
 getGenesisSummary = do
-    redeemAddressCoinPairs <- getRedeemAddressCoinPairs
-    cgsNumRedeemed <- length <$> filterM (uncurry isAddressRedeemed) redeemAddressCoinPairs
-    pure CGenesisSummary {cgsNumTotal = length redeemAddressCoinPairs, ..}
+    grai <- getGenesisRedeemAddressInfo
+    cgsNumRedeemed <- V.length <$> V.filterM (uncurry isAddressRedeemed) grai
```

Comment:
`length` in `Universum` is polymorphic, btw (maybe you know it already and prefer to be more explicit).

Reply:
Thank you, I didn't know that. Replaced `V.length` with `length`.